src/Microsoft.Diagnostics.Runtime/src/DacInterface/DacDataTargetWrapper.cs

```diff
@@ -254,30 +255,29 @@ public int Request(IntPtr self, uint reqCode, uint inBufferSize, IntPtr inBuffer
             if (filePath is null)
                 return E_FAIL;
 
-            // We do not put a using statement here to prevent needing to load/unload the binary over and over.
+            // We do not dispose the loaded file to avoid loading/unloading the same binary over and over.
             PEImage? peimage = _dataTarget.LoadPEImage(filePath);
             if (peimage is null || peimage.OptionalHeader is null)
                 return E_FAIL;
 
             DebugOnly.Assert(peimage.IsValid);
 
-            uint rva = mdRva;
-            uint size = bufferSize;
-            if (rva == 0)
+            Stream peStream = peimage.Stream;
```

Comment:
This change made me realize that none of the PEImage code is threadsafe...  I need to fix that, but it's not something you have to address in this change.